src/parser/jobs/mnk/modules/OGCDDowntime.ts

```diff
@@ -1,17 +1,38 @@
 import ACTIONS from 'data/ACTIONS'
 import {CooldownDowntime} from 'parser/core/modules/CooldownDowntime'
 
-// Approximate downtime window to account for aligning derped setups and GCD pinning:
-// 2 GCDs under slow GL3 (2.07s GCD), 700ms for second half of GCD activation
+// Approximate downtime window to account for aligning GCDs and unreliable FBC procs:
+// 2 GCDs at a slow 2.0s GCD, 600ms for animation lock, and another 250ms to deal with
+// high end of "sane" ping or a late FBC proc clipping the GCD.
 const DEFAULT_ALLOWED_DOWNTIME = 4850
```

Comment:
Given your comments, you need to reduce this number to 2425 -- if the allowable downtime window is 2GCDs plus the other acoutrements, the average value that CDDT should allow is half that (to account for times that skills get used right on time and thus have 0 downtime).

Reply:
Oh good find, I completely forgot this is an average.